deploy/crds/push_v1alpha1_unifiedpushserver_cr_with_external_db.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: push.aerogear.org/v1alpha1
+kind: UnifiedPushServer
+metadata:
+  name: example-unifiedpushserver
+spec:
+  useMessageBroker: false
+  externalDB: true
+  database: 
+    databaseHost: example-host
+    databaseName: example-name
+    databasePassword: password
+    databaseUser: user
+    databasePort: 5432
```

Comment:
I think it's a bit redundant to have "database" here twice. What do you think about changing it to this instead:

```yaml
  database: 
    host: example-host
    port: 5432
    name: example-name
    user: user
    password: password
```